common/graalvm-reachability-metadata/src/main/java/org/graalvm/reachability/internal/index/artifacts/SingleModuleJsonVersionToConfigDirectoryIndex.java

```diff
@@ -101,15 +103,18 @@ public Optional<DirectoryConfiguration> findLatestConfigurationFor(String groupI
     }
 
     /**
-     * Returns the latest configuration directory for the requested artifact.
+     * Returns the matching configuration directory for the requested artifact.
      *
      * @param groupId the group ID of the artifact
      * @param artifactId the artifact ID of the artifact
+     * @param version the version of the artifact
      * @return a configuration directory, or empty if no configuration directory is available
      */
     @Override
     public Optional<DirectoryConfiguration> findLatestConfigurationFor(String groupId, String artifactId, String version) {
```

Comment:
The name is no longer accurate. Not sure if we want to change it?

Reply:
Could you please elaborate?

Reply:
find**Latest**ConfigurationFor, shouldn't it be something like findMatchingConfigurationFor now?